### PR TITLE
[FIX] stability improvement on crash reporter

### PR DESCRIPTION
**Pull request by rksh**
collecting more info on some error-email-compose runtime exceptions, lowering the max size a little in hopes of preventing.

### CODE REVIEW

**Review by dangerzone69 — COMMENTED** [review]
Hmm

**Review by bobzilladev — APPROVED** [review]
Stack and Attack!

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java [hunk 1]
```diff
@@ -171,9 +171,11 @@ private void setupEmail(final String stack, final String stackFile ) {
         final Intent chooserIntent = Intent.createChooser( emailIntent, "Email WigleWifi error report?" );
         try {
             startActivity(chooserIntent);
-        }
-        catch (final ActivityNotFoundException ex) {
-            MainActivity.warn("No email activity found: " + ex, ex);
+        } catch (final ActivityNotFoundException ex) {
```
Comment (dangerzone69) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java:174]: Hello world

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
 public class ErrorReportActivity extends AppCompatActivity {
     // https://developer.android.com/reference/android/os/TransactionTooLargeException
     // ALIBI: this may still be too big in some cases, but we know we can't be bigger than the max - the size of the other data we package
-    public final static int MAX_STACK_TRANSACTION_SIZE = (1024 * 1024) - 512;
+    public final static int MAX_STACK_TRANSACTION_SIZE = (1024 * (1024-2));
```
Comment (dangerzone69) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java:31]: (Hello world)

**Review by dangerzone69 — COMMENTED** [review]
310410_20485_185757192

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java (hunk 1 above)
Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/ErrorReportActivity.java:174]: further nonsense comments will result in banning.

**Pull request merged**